### ISSUE TITLE
an error happened when import dlib and MxNet

### ISSUE THREAD

**Issue by PuNeal** [becauseofAI/MobileFace#13]
When I run the ./example/get_face_align.py, an error happened. The error meassage is:

Segmentation fault: 11

Stack trace returned 10 entries:
[bt] (0) /root/anaconda3/lib/python3.6/site-packages/mxnet/libmxnet.so(+0x40123a) [0x7f8a3cba723a]
[bt] (1) /root/anaconda3/lib/python3.6/site-packages/mxnet/libmxnet.so(+0x3413b76) [0x7f8a3fbb9b76]
[bt] (2) /lib64/libc.so.6(+0x36280) [0x7f8b08f7b280]
[bt] (3) /root/anaconda3/lib/python3.6/site-packages/onnx/onnx_cpp2py_export.cpython-36m-x86_64-linux-gnu.so(+0x2e69e) [0x7f8a1d80669e]
[bt] (4) /root/anaconda3/lib/python3.6/site-packages/onnx/onnx_cpp2py_export.cpython-36m-x86_64-linux-gnu.so(+0x38739) [0x7f8a1d810739]
[bt] (5) /root/anaconda3/lib/python3.6/site-packages/onnx/onnx_cpp2py_export.cpython-36m-x86_64-linux-gnu.so(+0x1ce83) [0x7f8a1d7f4e83]
[bt] (6) /root/anaconda3/lib/python3.6/site-packages/onnx/onnx_cpp2py_export.cpython-36m-x86_64-linux-gnu.so(PyInit_onnx_cpp2py_export+0xd6) [0x7f8a1d7f8946]
[bt] (7) python(_PyImport_LoadDynamicModuleWithSpec+0x185) [0x55583797ba45]
[bt] (8) python(+0x215c45) [0x55583797bc45]
[bt] (9) python(PyCFunction_Call+0x131) [0x555837878051]

It happened when importing mxnet and dlib, and I found that mxnet and dlib cannot be imported correctly in one python file. But it succeeded when importing mxnet or dlib alone. Could you tell me the version of mxnet and dlib installed in your environment ? 
Thank you!

**Comment by becauseofAI** [comment]
Can you check the problem to import mxnet and dlib through python command line firstly?

**Comment by PuNeal** [comment]
> Can you check the problem to import mxnet and dlib through python command line firstly?

Yes, I tried to import dlib and mxnet through python command line but failed. If i import mxnet  successfully at first, then dlib cannot be imported correctly, and vice versa. As the screenshot shows below:
<img width="826" alt="捕获" src="https://user-images.githubusercontent.com/39493003/58526239-e5e34b00-8200-11e9-9d1b-04552a7160ec.PNG">

**Comment by becauseofAI** [comment]
Strangely!
If you haven't fixed it yet, I think the easiest way is to reinstall Anaconda3, MXNet and Dlib in an official manner. You can reference my [README](https://github.com/becauseofAI/MobileFace/blob/master/README.md) to install MXNet and Dlib after you have installed Anaconda3.
